dependency-manager-tests/spm/SPMProject/DetailViewController.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  DetailViewController.swift
+//  SPMProject
+//
```

Comment:
Same here - license header. Also, please run `./tools/license/check-license.sh` to see if all is good.